src/reducers/AppState.ts

```diff
@@ -1,5 +1,5 @@
-import { AppStateV4 } from './version4';
+import { AppStateV5 } from './version5';
```

Comment:
It's not related to this PR, but wanted to raise this topic anyway. Not sure how much it matters from at app start, but we could merge the the version upgrades which happened between two versions. If nobody used 3 and 4, we can do the change in one run.

Reply:
Good catch!
I check usually between releases but our last release was almost two months ago, so I just assumed that we have to change, whereas we can still release as version4.